### ISSUE TITLE
Let's shorten our menu

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#15642]
Over time I have met many users who facing very basic issues by not realizing thatt our main menu **can and must be scrolled** to, on many smaller devices, even see such basic entries as Settings or Help. (Chances are this is also the root cause for  issues like #15639.)

Here's how things often look:
![Screenshot_20221104-112253_OsmAnd~](https://user-images.githubusercontent.com/845735/199951403-ff0d0367-9586-4e49-b0db-dc2aae45be7a.jpg)

We could remediate the situation by cleaning this up a bit to make it more discoverable, e.g. by
- consolidating what are Settings (e.g. all the "Configere profile/map/screen) under one top level Settings entry (#9047)
- other than that single entry, keep only app **Functions**("actions") in that list.

**Comment by franco999** [comment]
Currently, OsmAnd has such a high level of customization that one often gets lost among so many menus. An additional option could be to implement an option search engine as Android settings already do.

I don't know how to code this and if it is really feasible here.

**Comment by sonora** [comment]
PS: In addition, the prominent header section of the menu drawer (The "Browse map" and "Cofigure profile" lines in the screenshot above) is both misleading and wastes screen real estate: You would expect that if you have a profile selector on the very top, this actually facilitates to switch the profile the app/map uses. It does not, instead it simply selects the profile for a subsequent "Configure profile" action (if any).

But that should be turned around: Simply have only an entry "Configure profiles" on the drawer, and on its subsequent screen have a selector at the top facilitating to pick the one to be configured (defaulting to the current one)..

**Comment by comradekingu** [comment]
My main problem was always remembering where the different settings are.

Set up profile
Map settings
Screen settings
Settings - General settings
              - profile settings again?

Search could be all the way at the top, with a gear icon next to the profile icon to configure it.
"Markers" and "Places" as two buttons in one entry.

Map settings and downloads could be "Maps".

"Navigation" and "Plan route" in one?

Navigation above tracks and travel guides at the bottom for clarity?

"Help" could be in settings, and "Build versions" could be in there too.

That is already enough it would seem, but the bigger issue is to do the settings a bit differently.

**Comment by vshcherb** [comment]
My take would be different 

1. Could be hidden cause present on the main screen as button
> Search 
> Navigation
> Configure map

2. Could be hidden cause accessible through Settings / Configure Profile
> Configure map
> Configure screen

3. Could be **migrated into Settings**:
> Plugins 
> Help 

So only 3rd requires some UI actions, other could be done in the code easily.

**Comment by sonora** [comment]
It's true, we could hide all actions usually accessed via the map screen directly.

I tend to agree,  but we have to be aware that this reduces discoverability: Some new users not aware of the meaning and use of screen widgets may expect basic functionality to be accessible via some sort of explicit text menu in the first place...

**Comment by vshcherb** [comment]
Yes, main screen is question whether we should hide all accessible and keep it configurable or not. It's definitely a bad question to ask experienced users cause they a) could customize the list b) need other functions.

We need to take into account that most of the users won't recustomize and they still will need sporadically change widgets or customize. So the biggest question "Comfortable to read in the beginning" vs "Comfortable to use later". I think the picture is not so much relevant cause most of the phones are larger but the question that people could be frighten that there are 4-5 menu items of configuration. 

So to be honest I don't see that we need to do smth. In the end we're discussing to hide only 3 items:
- Configure map
- Configure screen
- Plugins

**Comment by sonora** [comment]
Yes, I guess if we could simply consolidate them in this fashion

<img width="170" alt="Screenshot 2022-11-10 133738_" src="https://user-images.githubusercontent.com/845735/201094423-4b471d89-d24f-4d9f-b456-0930d26f19f8.png">
.
that alone could be a quick win and a huge step forward
